connectors/protocol/connectors.py

```diff
@@ -596,6 +597,8 @@ def _property_as_datetime(self, key):
         value = self.get(key)
         if value is not None:
             value = parse_datetime_string(value)  # pyright: ignore
+            # ensure that the datetime is in UTC timezone
```

Comment:
I think it's also worth to add a comment here mentioning the problem found and saying that it's done here for backwards-compatibility

Reply:
Okok, added the comment!